plugin.yml

```diff
@@ -15,6 +15,9 @@ configuration:
     stack:
       description: Name of the stack to operate on
       type: string
+    refresh:
+      description: Should Pulumi perform a refresh before the update/preview? ("true" or "false")
+      type: string
```

Comment:
If you declare this as an actual boolean type, would that work better? Alternatively, if it's a string enum of `["true", "false"]`, will it even let you pass "waffles" as a value?